mypy/build.py

```diff
@@ -1012,6 +1038,12 @@ class State:
     # If caller_state is set, the line number in the caller where the import occurred
     caller_line = 0
 
+    # If True, indicate that the public interface of this module is unchanged
+    externally_same = True
+
+    # Contains a hash of the public interface in incremental mode
+    interface_hash = "never_set"  # type: str
```

Comment:
Does "never_set" have a special meaning? It doesn't seem to. I'd just use an empty string if the sole purpose is to make it type-check correctly.